incubator/blockchain-client-openliberty/templates/default/README.md

```diff
@@ -0,0 +1,96 @@
+# Default template for Blockchain with Open Liberty
+
+## Introduction
+
+The intent of the default template in the Blockchain Open Liberty stack is to provide a working example that easily integrates with the default Smart Contract provided by The IBM® Blockchain Platform Visual Studio (VS) Code extension.(<https://cloud.ibm.com/docs/services/blockchain/howto?topic=blockchain-develop-vscode)>
+
+That default Smart Contract is called MyAsset and following a minimal set of instructions detailed in the link above you can have a Fabric Network running the MyAsset Smart Contract.  This template provides an means to easily bind a Java Openlibery Service to the MyAsset Smart Contract. For learning purposes your new project is able to run, connect and execute transactions without any code changes. Not only can you bind to a Local Fabric Instance created by VSCode, you can in fact interact with any Blockchain Fabric Network instance.(<https://www.ibm.com/blockchain/platform)>
```

Comment:
`This template provides an means...` -> `This template provides a means...`

Reply:
`easily bind a Java Openlibery Service` -> `expose a JAX-RS REST service` (perhaps?)

Reply:
Thanks will make those changes.